backend/controller/controller.go

```diff
@@ -1074,6 +1072,34 @@ func (s *Service) callWithRequest(
 		}
 	}
 
+	callEvent := &timeline.Call{
+		DeploymentKey:    route.Deployment,
+		RequestKey:       requestKey,
+		ParentRequestKey: parentKey,
+		StartTime:        start,
+		DestVerb:         verbRef,
+		Callers:          callers,
+		Request:          req.Msg,
+	}
+
+	if currentCaller != nil && currentCaller.Module != module && !verb.IsExported() {
+		observability.Calls.Request(ctx, req.Msg.Verb, start, optional.Some("invalid request: verb not exported"))
+		err = connect.NewError(connect.CodePermissionDenied, fmt.Errorf("verb %q is not exported", verbRef))
```

Comment:
Since this is the same as the error you return, should this just be:
```
err = connect.NewError(connect.CodePermissionDenied, fmt.Errorf("verb %q is not exported", verbRef))
callEvent.Response = either.RightOf[*ftlv1.CallResponse](err)
s.timeline.EnqueueEvent(ctx, callEvent)
return nil, err
```

Reply:
Nice, yeah good call. Updated.